src/time.cr

```diff
@@ -452,7 +474,8 @@ struct Time
     epoch.to_f + nanosecond.to_f / 1e9
   end
 
-  def to_utc
+  # Converts this time to UTC.
```

Comment:
Maybe "converts" can be understood as if it mutates the instance, maybe something like "Returns a new Time instance where ..." but I don't know how to reword it.